### DISCUSSION TITLE
nixos password?

### DISCUSSION THREAD

**Discussion by PlumpClojure** [nix-community/NixOS-WSL#71] [Q&A] [answered]
.

**Comment by nzbr** [comment, marked as answer]
Per default, the nixos user does not have a password. You should be able to use sudo without one. You may change the password with `sudo passwd nixos`, after that you can set `security.sudo.wheelNeedsPassword = true` if you want to be prompted for it when using sudo.

I hope, that this answered your question. If not, [please provide additional details about your problem](https://stackoverflow.com/help/how-to-ask)
